### PR TITLE
Create portfolio layout

### PR DESCRIPTION
**Pull request by seb-lex**
Udacity project.

# Prototype:
image

# Result:
image

### CODE REVIEW

**Review by Martian2Lee — CHANGES_REQUESTED** [review]
not best practice, please update

**Comment by sebz996** [comment]
Done.

**Comment by seb-lex** [comment]
Done. 

image

File: portfolio/css/style.css [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    display: flex;
+}
+
+header {
+    height: 140px;
+    border-bottom: 2px solid black;
+    margin-bottom: 20px;
+}
+
+header div {
```
Comment (Martian2Lee) [inline portfolio/css/style.css]: bad identifier, ask me to explain in the call

File: portfolio/css/style.css [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+main {
+    max-width: 800px;
+    margin: 0 auto;
+    font-family: 'Roboto';
+}
+
+section {
+    display: flex;
+    justify-content: space-between;
```
Comment (Martian2Lee) [inline portfolio/css/style.css:13]: 👍

**Review by Martian2Lee — CHANGES_REQUESTED** [review]
Done.

File: portfolio/css/style.css (hunk 2 above)
Comment (seb-lex) [reply portfolio/css/style.css:13]: Changes requested:

image

**Review by Martian2Lee — APPROVED** [review]
🚀

**Pull request merged**